thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/views/diffsummary/SummaryResponseTree.java

```diff
@@ -32,7 +33,8 @@
 
     List<SummaryResponseTreeNode> treeNodes = new ArrayList<>();
     // Build the response tree
-    nodes.sort(Summary.NODE_COMPARATOR.reversed()); // pre-order traversal
+    Collections.sort(nodes, Summary.NODE_COMPARATOR); // pre-order traversal
+    Collections.reverse(nodes);
```

Comment:
Can be combined as Collections.sort(nodes, Collections.reverse(Summary.NODE_COMPARATOR));